### PR TITLE
Preview atoms

### PR DESCRIPTION
**Pull request by regiskuckaertz**
This is a very first shot at a way to preview atoms quickly to speed up development.

Todo
- [x] Add some sensible typography defaults
- [x] Add CSS variables
- [x] Update docs

screen shot 2019-02-18 at 09 14 33

### CODE REVIEW

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  val client = new GuardianContentClient(sys.env("CAPI_TEST_KEY"))
+
+  val assetsService: HttpRoutes[IO] = HttpRoutes.of[IO] {
+    case req @ GET -> Root / "styles" ~ "css" =>
+      StaticFile.fromResource("/styles.css", global, Some(req)).getOrElseF(NotFound())
+  }
+
+  val capiService: HttpRoutes[IO] = HttpRoutes[IO] {
+    case GET -> Root :? AtomTypeMatcher(atomType) +& AtomIdMatcher(atomId) =>
```
Comment (ajwl) [inline utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: Hmm I find Scala syntax like this very hard to understand. Perhaps you can talk me through this on Monday?

Comment (regiskuckaertz) [reply utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: Yes of course

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  val client = new GuardianContentClient(sys.env("CAPI_TEST_KEY"))
+
+  val assetsService: HttpRoutes[IO] = HttpRoutes.of[IO] {
+    case req @ GET -> path =>
+      StaticFile.fromResource(s"$path", global, Some(req)).getOrElseF(NotFound())
+  }
+
+  val capiService: HttpRoutes[IO] = HttpRoutes[IO] {
+    case GET -> Root :? AtomTypeMatcher(atomType) +& AtomIdMatcher(atomId) =>
+      OptionT.liftF(IO.async[ItemResponse] { cb =>
```
Comment (JustinPinner) [inline utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: Can we use more meaningful names than `cb`, `t` and `r`?

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          case AtomType.Timeline => 
+            resp.timeline.fold(NotFound(s"Audio atom with id $atomId could not be found"))(a => 
+              Ok(html.page(
+                Some(a),
+                Some(Html(ArticleAtomRenderer.getHTML(a, config))),
+                ArticleAtomRenderer.getCSS(AtomType.Timeline),
+                ArticleAtomRenderer.getJS(AtomType.Timeline)
+              ))
+            )
+          case _                 => InternalServerError("Hmm, this is weird")
```
Comment (JustinPinner) [inline utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: ("Hmm, this is weird - " + `atomType` + ' is not what we expected")

File: utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      })
+
+    case GET -> Root =>
+      OptionT.liftF(Ok(html.page(None, None, None, None)))
+
+  }
+
+  def run(args: List[String]): IO[ExitCode] =
+    BlazeServerBuilder[IO]
+      .bindHttp(9000, "localhost")
```
Comment (JustinPinner) [inline utils/src/main/scala/com.gu.contentatom.renderer/utils/CapiService.scala]: `9000` will clash with dotcom frontend if that's running I think?

**Comment by regiskuckaertz** [comment]
@JustinPinner 👍 I've gone through your feedback.

**Pull request merged**